aspnetcore/test/troubleshoot-azure-iis.md

```diff
@@ -0,0 +1,607 @@
+1. Select **Advanced Settings** under **Edit Application Pool** in the **Actions** panel.
+1. Set **Enable 32-Bit Applications**:
+   * If deploying a 32-bit (x86) app, set the value to `True`.
+   * If deploying a 64-bit (x64) app, set the value to `False`.
+
+### Connection reset
+
+If an error occurs after the headers are sent, it's too late for the server to send a **500 Internal Server Error** when an error occurs. This often happens when an error occurs during the serialization of complex objects for a response. This type of error appears as a *connection reset* error on the client. [Application logging](xref:fundamentals/logging/index) can help troubleshoot these types of errors.
+
+### Default startup limits
+
+The [ASP.NET Core Module](xref:host-and-deploy/aspnet-core-module) is configured with a default *startupTimeLimit* of 120 seconds. When left at the default value, an app may take up to two minutes to start before the module logs a process failure. For information on configuring the module, see [Attributes of the aspNetCore element](xref:host-and-deploy/aspnet-core-module#attributes-of-the-aspnetcore-element).
+
+## Troubleshoot on Azure App Service
+
+[!INCLUDE [Azure App Service Preview Notice](~/includes/azure-apps-preview-notice.md)]
+
+### Application Event Log (Azure App Service)
+
+To access the Application Event Log, use the **Diagnose and solve problems** blade in the Azure portal:
+
+1. In the Azure portal, open the app in **App Services**.
+1. Select **Diagnose and solve problems**.
+1. Select the **Diagnostic Tools** heading.
+1. Under **Support Tools**, select the **Application Events** button.
+1. Examine the latest error provided by the *IIS AspNetCoreModule* or *IIS AspNetCoreModule V2* entry in the **Source** column.
+
+An alternative to using the **Diagnose and solve problems** blade is to examine the Application Event Log file directly using [Kudu](https://github.com/projectkudu/kudu/wiki):
+
+1. Open **Advanced Tools** in the **Development Tools** area. Select the **Go&rarr;** button. The Kudu console opens in a new browser tab or window.
+1. Using the navigation bar at the top of the page, open **Debug console** and select **CMD**.
+1. Open the **LogFiles** folder.
+1. Select the pencil icon next to the *eventlog.xml* file.
+1. Examine the log. Scroll to the bottom of the log to see the most recent events.
+
+### Run the app in the Kudu console
+
+Many startup errors don't produce useful information in the Application Event Log. You can run the app in the [Kudu](https://github.com/projectkudu/kudu/wiki) Remote Execution Console to discover the error:
+
+1. Open **Advanced Tools** in the **Development Tools** area. Select the **Go&rarr;** button. The Kudu console opens in a new browser tab or window.
+1. Using the navigation bar at the top of the page, open **Debug console** and select **CMD**.
+
+#### Test a 32-bit (x86) app
+
+##### Current release
```

Comment:
h4-h5 is stretching the limits of what can be perceived in terms of different heading levels.  Probably something to be addressed at some other time, but it might be possible to elevate some h3s to h2s or otherwise flatten the structure.

Reply:
I did once. On the first pass, I had **sixth level headings** after pasting the content all in here from the two topics. 🙈:smile:

One thing we can do is resort to the use of bold as we do elsewhere in similar situations ...

```
**Current release**
```

... works just as well and looks pretty good when rendered.

I'll go thru and see how the fifth level headings can be dropped.